home-page.html

```diff
@@ -11,11 +11,63 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap" rel="stylesheet">
+    <script src="script.js" defer></script>
+    <script src="contributors.js" defer></script>
+    <link rel="stylesheet" href="pop-up-menu.css">
 </head>
-<body>
+<body id="home-page">
     <section class="background-image-wrapper">
         <div class="over-lay">
-            <header class="about-mobile-head"><i class="fa-sharp fa-solid fa-bars"></i></header>
+            <header class="about-mobile-head"><i class="fa-sharp fa-solid fa-bars"></i>
+                <div id="pop-up-menu-wrapper">
+                    <i class="fa-solid fa-xmark" id="cancel"></i>
+                <div class="pop-up-menu">
+                    <ul class=".pop-up-list">
+                        <li class="pop-up-li"><a class="page-link" href="home-page.html">Home</a></li>
+                        <li class="pop-up-li"><a class="page-link" href="index.html">About</a></li>
+                        <li class="pop-up-li">Program</li>
+                        <li class="pop-up-li">Join</li>
+                        <li class="pop-up-li">Sponsor</li>
+                    </ul>
+                </div>
+                </div></header>
+            <header class="about-desk-head">
+                <div class="sign-in-bar">
+                    <i class="fa-brands fa-facebook"></i>
+                    <i class="fa-brands fa-twitter"></i>
+                    <ul>
+                        <li><a href="">Logout</a></li>
+                        <li><a href="">Mypage</a></li>
+                        <li><a href="">Extras</a></li>
+                    </ul>
+                </div>
+                <div class="white-part">
+                    <div class="cc-image">
+                        <i class="fa-solid fa-closed-captioning"></i>
+                    </div>
+                    <div class="text">
+                        <p>
+                            CC GLOBAL
+                        </p>
+                        <p>
+                            Summit 2015
+                        </p>
+                        <p class="part-two dates">
+                            14th - 17th oct 2015 | Seoul, Korea
+                        </p>
+                    </div>
+                    <div class="menu-list">
+                        <ul class="menu-list-list">
+                            <li><a href="index.html">About</a></li>
+                            <li><a href="">Program</a></li>
```

Comment:
i feel like youre overscrutinizing here and for 30 mins fitting all these changes in when i made the review an 3 hours ago

Reply:
@Reem-lab

Reply:
also try to be more specific tell me the problem instead of just drawing lines

Reply:
Hello @stino-x I have to check well to help improve your project, I just spent the estimated time for the review.
I explained well the problem above in my comments but let me repeat again I think you need  first of all It would be great if you align your items to be centered like the original design so kindly fix that ✨💯🚀 so I drawed lines to let you know that you need to align all your items to be centered like the given design. ✨🔥 I hope if anything is not clear tell me to expand my explaniation 🙏🏻😊

Reply:
@stino-x  and please read the comments well above the images I think I explained well what you should update 😊